inc/class-wpseo-utils.php

```diff
@@ -1104,6 +1104,24 @@ public static function get_admin_l10n() {
 		return $wpseo_admin_l10n;
 	}
 
+	/**
+	 * Retrieves the analysis worker log level. Defaults to errors only.
+	 *
+	 * Uses bool YOAST_SEO_DEBUG as flag to enable logging. Off equals ERROR.
+	 * Uses string YOAST_SEO_DEBUG_ANALYSIS_WORKER as log level for the Analysis
+	 * Worker. Defaults to INFO.
+	 * Can be: TRACE, DEBUG, INFO, WARN or ERROR.
+	 *
+	 * @return string The log level to use.
+	 */
+	public static function get_analysis_worker_log_level() {
```

Comment:
Then this function could return an PHP array of log levels.
In this case with only one entry for `analyis_worker`.